.github/workflows/e2e-test-weekly.yml

```diff
@@ -241,6 +241,12 @@ jobs:
           kubernetesVersion: ${{ matrix.kubernetes-version }}
           provider: ${{ matrix.provider }}
 
+      - name: Always upload Terraform logs
+        if: always()
+        uses: ./.github/actions/upload_terraform_logs
+        with:
+          artifactNameSuffix: ${{ steps.e2e_test.outputs.namePrefix }}
```

Comment:
Can't we do this step on a lower layer, like in constellation_destroy? Otherwise lgtm.

Reply:
I think it'd be a bad practice to put it in either constellation_destroy or constellation_iam_destroy. If you put it in 1 of the 2 you would implicitly require that action to be always executed last (to not lose log output)